web/templates/message_feed_errors.hbs

```diff
@@ -12,25 +12,21 @@
 <div class="all-messages-search-caution hidden-for-spectators" hidden>
     <p>
         {{#tr}}
-        End of results from your
-        <z-link>history</z-link>.
-        {{#*inline "z-link"}}<a href="/help/search-for-messages#search-shared-history"
+        End of results from
+        <z-link>your message history</z-link>.
+        {{#*inline "z-link"}}<a class="search-shared-history" href=""
           target="_blank" rel="noopener noreferrer">{{> @partial-block}}</a>{{/inline}}
         {{/tr}}
         &nbsp;
-        <span>
+    </p>
+    <div class="search-all-channels-button">
+        <button class="search-public-channels-btn action-button action-button-quiet-neutral" type="button">
             {{#if is_guest}}
-                {{#tr}}
-                    Consider <z-link>searching all public channels that you can view</z-link>.
-                    {{#*inline "z-link"}}<a class="search-shared-history" href="">{{> @partial-block}}</a>{{/inline}}
-                {{/tr}}
+                Search all public channels that you can view
```

Comment:
Hmmm... does the issue or any of the design discussions note what the button text should be for guests? This seems a bit long.